### ISSUE TITLE
Missing .docker-stack in tutum-phundament-eco

### ISSUE THREAD

**Issue by drsdre** [neam/docker-stack#6]
.docker-stack is missing in tutum-phundament-eco:

```
→ docker-stack select tutum-phundament-eco
cat: ../../docker-stack/stacks/tutum-phundament-eco/.docker-stack: No such file or directory
Stack 'tutum-phundament-eco' selected
```

**Comment by schmunk42** [comment]
@motin My stack, your CLI - we have to check together what's going wrong.

**Comment by drsdre** [comment]
I would have expected a .docker-stack in the directory https://github.com/neam/docker-stack/tree/develop/stacks/tutum-phundament-eco, but don't see one. Or is it supposed to be auto-generated?

**Comment by motin** [comment]
Yes, to be able to select an existing stack without installing it, the .docker-stack file needs to be present. "tutum-phundament-eco" needs to be installed since it is missing this file.

**Comment by schmunk42** [comment]
@drsdre btw: You can simply run `docker-compose up` with the file from the directory.

It is meant to save resources in development or staging environments, by having just one container from the MySQL database and redis, along with one reverse proxy for virtual hosts.
